src/main/java/Models/Appointment/Appointment.java

```diff
@@ -61,6 +63,18 @@ public String getType() {
         return type;
     }
 
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public static String getDayOfDate(Date date) {
```

Comment:
This is a reminder to myself to update all Appointment unit tests, as they will all be broken now.